modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLSizeLimitConfigurationProvider.java

```diff
@@ -15,6 +15,8 @@
 @ProviderType
 public interface DLSizeLimitConfigurationProvider {
 
+	public boolean canCopy(long dlFolderSize, long groupId, long companyId);
```

Comment:
change name of dLFolderSize, not always a folder.

Reply:
And...

I'm not sure about the name :S

Reply:
We will go with `isCopyToAllowed`